src/free-trial/tax/woocommerce-tax/setup.tsx

```diff
@@ -101,25 +102,14 @@ export const Setup: React.FC< SetupProps > = ( {
 			),
 			content: <StoreLocation { ...stepProps } />,
```

Comment:
I previously changed all the fields to mandatory in [this Core PR](https://github.com/woocommerce/woocommerce/pull/38137) because [all fields are required to enable tax automation](https://github.com/Automattic/wc-calypso-bridge/blob/520f3e6de237f3556f0ab64d57b40648e9347a57/src/free-trial/tax/woocommerce-tax/index.tsx#L53). I think we should also modify it here, but I'm okay if we do it in a follow-up PR.